common/src/main/java/org/opensearch/ml/common/MLModel.java

```diff
@@ -143,6 +143,42 @@ public class MLModel implements ToXContentObject {
     private String connectorId;
     private Guardrails guardrails;
 
+    /**
+     * Model interface is a map that contains the input and output fields of the model, with JSON schema as the value.
```

Comment:
Can user just define input or output ?

Reply:
Yes, user can define it separately, and check will start separately. No need to define all